assets/data/cards_pt-br.json

```
@@ -296,6 +296,21 @@
       "title": "Pull Request",
       "description": "Trata-se de uma requisição de alteração dentro de um projeto. Essa requisição pode ser feita para corrigir um problema, adicionar uma nova funcionalidade, sugerir alterações, entre outros motivos. Essa requisição de alteração é feita para que haja uma avaliação do que está sendo alterado dentro do projeto, entendendo de faz sentido para o contexto daquele momento. E essa requisição pode ficar pendente até que pessoas responsáveis ou que tenham a permissão, aprovem. Outras regras podem ser adicionadas nessas requisições, como build da aplicação, rodar testes com sucesso e o que mais fizer sentido para o projeto em questão.",
       "tags": ["Conceito"]
-    }    
+    },
+    {
+      "title": "Garbage Collector",
+      "description": "Literalmente \"coletor de lixo\" em inglês, é um tipo de software auxiliar que é executado ao lado de uma aplicação principal, percebendo memória que foi alocada (designada ao programa) mas não é mais necessária, permitindo que seja retornada ao sistema operacional. Geralmente, é usado por determinação conta da linguagem utilizada, como Java, C# e Javascript",
```

Sugiro que expandamos um pouco mais a descrição para torná-la mais detalhada e acessível. Isso pode ajudar a esclarecer o conceito para quem não está familiarizado com ele. Abaixo, deixei uma sugestão de como podemos enriquecer o texto para tornar a explicação mais clara e compreensiva.

```suggestion
      "description": "Garbage collector, ou \"coletor de lixo\", é uma ferramenta que ajuda a liberar espaço na memória do computador automaticamente. Imagine que, ao desenvolver um programa, você use blocos de memória para guardar informações. Com o tempo, alguns desses blocos não são mais necessários. O coletor de lixo identifica esses blocos "inúteis" e os limpa, devolvendo o espaço para o sistema usar novamente. Isso é especialmente útil em linguagens de programação como Java, C# e JavaScript, onde esse processo é feito automaticamente, ajudando a evitar problemas e deixando o computador mais rápido.",
```